questionnaire/test-fixtures/res/get_questionnaire.json

```diff
@@ -620,7 +620,8 @@
                 "version": "0.1.0",
                 "progress": [
                     "p-applicant-declaration",
-                    "p-applicant-british-citizen-or-eu-national"
+                    "p-applicant-british-citizen-or-eu-national",
```

Comment:
Is this used?

Maybe do these changes in a "completed questionnaire" fixture rather than a partially completed fixture